packages/core/src/components/essentials/tooltip.ts

```diff
@@ -145,6 +148,8 @@ export class Tooltip extends Component {
 		if (type === TooltipTypes.TITLE) {
 			const title = this.model.getOptions().title;
 			return `<div class="title-tooltip"><text>${title}</text></div>`;
+		} else if (type === TooltipTypes.LEGEND) {
+			return `<div class="title-tooltip"><text>${data.name}</text></div>`;
```

Comment:
```suggestion
			return `<div class="legend-tooltip"><text>${data.name}</text></div>`;
```
in case users want to target the classes separately

Reply:
or we can have a truncation tooltip class so we wont need to change this to 
```
		.title-tooltip {
			line-height: 16px;
			font-size: 12px;
			padding: 4px;
			min-width: 20px;
		}
```
to this 

```
		.title-tooltip, .legend-tooltip, .axis-tooltip {
			line-height: 16px;
			font-size: 12px;
			padding: 4px;
			min-width: 20px;
		}

```